docs/open-in-editor.md

```diff
@@ -6,30 +6,34 @@ Nuxt supports this feature out-of-the-box. Make sure to be in debug mode.
 
 ## Webpack
 
-In your Vue project, install the [express-open-in-editor](https://github.com/lahmatiy/express-open-in-editor) package and modifiy your webpack configuration:
+In your Vue project, install the [launch-editor-middleware](https://github.com/yyx990803/launch-editor#middleware) package and modifiy your webpack configuration:
 
 1. Import the package:
 
 ```js
-var openInEditor = require('express-open-in-editor')
+var openInEditor = require('launch-editor-middleware')
 ```
 
 2. In the `devServer` option, register the `/__open-in-editor` HTTP route:
 
 ```js
 devServer: {
   before (app) {
-    app.use('/__open-in-editor', openInEditor({
-      editor: 'code'
-    }))
+    app.use('/__open-in-editor', openInEditor())
   }
 }
 ```
 
-3. You can change `'code'` with the editor you are using, see the [supported editors list](https://github.com/lahmatiy/open-in-editor#editor).
+3. The editor to launch is guessed. You can also specify the editor app with the `editor` option. See the [supported editors list](https://github.com/lahmatiy/open-in-editor#editor).
```

Comment:
Supported editors was  added in the launch-editor package.

https://github.com/yyx990803/launch-editor#supported-editors